cmd/healthcheck.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/miekg/dns"
+	"github.com/spf13/cobra"
+)
+
+const (
+	defaultDNSPort = 53
+)
+
+func NewHealthcheckCommand() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "healthcheck",
+		Short: "performs healthcheck",
+		RunE:  healthcheck,
+	}
+
+	c.Flags().Uint16P("port", "p", defaultDNSPort, "healthcheck port 5333")
+
+	return c
+}
+
+func healthcheck(cmd *cobra.Command, args []string) error {
```

Comment:
Maybe we should add some log output here (like OK/NOT), since this command can be executed by user manually?

Reply:
added OK & NOT OK as healthcheck output